lab/src/Microsoft.ServiceModel.Syndication/tests/TestFeeds/feedWrongDate.xml

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<rss xmlns:a10="http://www.w3.org/2005/Atom" version="2.0">
+  <channel xml:base="http://mypage.com/">
+    <title>Microsoft News</title>
+    <link>http://www.microsoft.com/news</link>
+    <description>&lt;div&gt;Most recent news from Microsoft&lt;/div&gt;</description>
+    <managingEditor>jerry@microsoft.com</managingEditor>
+    <lastBuildDate>Tue, 23 August 2016 16:08:00 EDT</lastBuildDate>
+    <image>
+      <url>http://2.bp.blogspot.com/-NA5Jb-64eUg/URx8CSdcj_I/AAAAAAAAAUo/eCx0irI0rq0/s1600/bg_Microsoft_logo3-20120824073001907469-620x349.jpg</url>
+      <title>Microsoft News</title>
+      <link>http://www.microsoft.com/news</link>
+    </image>
+    <a10:id>123FeedID</a10:id>
+    <CustomElement>asd</CustomElement>
+    <item>
+      <link>http://microsoft.com/news/path</link>
+      <title>SyndicationFeed released for .net Core</title>
+      <description>A lot of text describing the release of .net core feature</description>
+    </item>
+  </channel>
+</rss>
```

Comment:
This file is not being used in the initial branch but I'm using it in the other one, will delete it.